plugin-server/src/utils/db/db.ts

```diff
@@ -143,6 +143,9 @@ export class DB {
     /** How many unique group types to allow per team */
     MAX_GROUP_TYPES_PER_TEAM = 5
 
+    /** Whether to write to clickhouse_person_unique_id topic */
```

Comment:
Let's keep writing to both tables for now, makes it easier to revert to reading from the old table in case we need it. I'm assuming it's not that big of a performance hit to write to both for a bit longer?

Reply:
We only deploy this when we are stable on cloud.

What is the failure scenario you are afraid of? The new table already powers a lot of reads and has proven itself stable. 

We can also always restore from psql which is a much better data source than anything else laying around in ch.

Reply:
Thinking more about self hosted failing in a weird yet unexpected way, but the restore from psql is a better way to restore it. On that note that might need to be in the rollback code before we allow self-hosted users to run that migration.